sdk/storage/src/lib.rs

```diff
@@ -9,6 +9,8 @@ extern crate serde_derive;
 extern crate azure_core;
 #[macro_use]
 extern crate quick_error;
+#[macro_use]
```

Comment:
Can we not use `macro_use` here? The preferred style in Rust 2018 is to use it like a regular item `lazy_static::lazy_static!`

Reply:
Removed since we are using `once_cell`!